metrics/lib/features/dashboard/domain/usecases/get_build_metrics.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:metrics/core/usecases/usecase.dart';
+import 'package:metrics/features/dashboard/domain/entities/build.dart';
+import 'package:metrics/features/dashboard/domain/repositories/metrics_repository.dart';
+import 'package:metrics/features/dashboard/domain/usecases/parameters/project_id_param.dart';
+
+/// Use case to load the build metrics
+class GetBuildMetrics implements UseCase<List<Build>, ProjectIdParam> {
+  final MetricsRepository _repository;
+
+  GetBuildMetrics(this._repository);
+
+  @override
+  Future<List<Build>> call(ProjectIdParam param) {
+    return _repository.getProjectBuilds(param.projectId);
+  }
+}
```

Comment:
Let's add newlines to the end of the files. Sometimes your IDE will have a setting to handle this automatically.

![image](https://user-images.githubusercontent.com/1173057/73586028-c7eb7b80-445c-11ea-9182-b89340e34ad1.png)